Source/KSCrash/Source/KSCrash/Recording/BSG_KSCrash.m

```diff
@@ -356,7 +357,17 @@ - (void)reportUserException:(NSString *)name
            terminateProgram:(BOOL)terminateProgram {
     const char *cName = [name cStringUsingEncoding:NSUTF8StringEncoding];
     const char *cReason = [reason cStringUsingEncoding:NSUTF8StringEncoding];
+    NSArray *addresses = [exception callStackReturnAddresses];
+    NSUInteger numFrames = [addresses count];
+    uintptr_t *callstack = malloc(numFrames * sizeof(*callstack));
+    for (NSUInteger i = 0; i < numFrames; i++) {
```

Comment:
How does this perform with very large stacktraces e.g. recursion gone wrong? The previous implementation has a limit of 100 frames.

Reply:
Unlimited stack errors can’t be caught and notified. This implementation is more in line with uncaught NSException handling.